.github/workflows/patch-release.yml

```diff
@@ -0,0 +1,115 @@
+name: Generate patch releases
+
+on:
+  workflow_dispatch:
+  schedule:
+    # First day of every month
+    - cron: '0 0 1 * *'
```

Comment:
is it right that a new release won't be created if there is no change in opencl-clang branch?

Reply:
Yes, that's correct